### PR TITLE
Handling CLI-only package error better

### PR DESCRIPTION
**Pull request by mlunoe**
This PR handles packages that are only installable through the CLI a little better and being more descriptive about what is happening.

Before:

After:

### CODE REVIEW

File: src/js/components/modals/InstallPackageModal.js [hunk 1]
```diff
@@ -258,15 +259,13 @@ class InstallPackageModal extends mixin(InternalStorageMixin, TabsMixin, StoreMi
             wrapperClass="" />
         </div>
         <div className="modal-footer">
-          <div className="container">
```
Comment (mlunoe) [inline src/js/components/modals/InstallPackageModal.js]: Removing the `container` div, which is not necessary

File: src/js/components/modals/InstallPackageModal.js [hunk 2]
```diff
@@ -513,14 +504,44 @@ class InstallPackageModal extends mixin(InternalStorageMixin, TabsMixin, StoreMi
...
+              onClick={this.handleModalClose}>
+              Close
+            </button>
+          </div>
+        </div>
+      </div>
+    );
+  }
+
+  getCLIPackageInfo(cosmosPackage) {
```
Comment (mlunoe) [inline src/js/components/modals/InstallPackageModal.js]: Only this package display is new

File: src/js/components/modals/InstallPackageModal.js [hunk 3]
```diff
@@ -532,6 +553,11 @@ class InstallPackageModal extends mixin(InternalStorageMixin, TabsMixin, StoreMi
     let {isLoading} = this.internalStorage_get();
     let cosmosPackage = CosmosPackagesStore.getPackageDetails();
 
+    // CLI only package
+    if (cosmosPackage && !cosmosPackage.getMetadata()) {
```
Comment (jfurrow) [inline src/js/components/modals/InstallPackageModal.js]: I'd suggest creating a method on the `UniversePackage` struct called `isCLIOnly` or something like that.

File: src/js/components/modals/InstallPackageModal.js [hunk 4]
```diff
@@ -513,14 +504,44 @@ class InstallPackageModal extends mixin(InternalStorageMixin, TabsMixin, StoreMi
...
+          <div className="modal-content-inner modal-content-inner-tall horizontal-center">
+            <div className="icon icon-jumbo icon-image-container icon-app-container">
+              <Image
+                fallbackSrc={defaultServiceImage}
+                src={cosmosPackage.getIcons()['icon-large']} />
             </div>
+            <p className="h2 short-top short-bottom">
+              {cosmosPackage.getName()}
+            </p>
+            <p>CLI only package</p>
```
Comment (jfurrow) [inline src/js/components/modals/InstallPackageModal.js]: This seems like a heading, should we capitalize it as such? `CLI Only Package`

File: src/js/components/modals/InstallPackageModal.js (hunk 3 above)
Comment (mlunoe) [inline src/js/components/modals/InstallPackageModal.js]: Done

File: src/js/components/modals/InstallPackageModal.js (hunk 4 above)
Comment (mlunoe) [inline src/js/components/modals/InstallPackageModal.js]: Done

File: src/js/components/modals/InstallPackageModal.js [hunk 5]
```diff
@@ -532,6 +553,11 @@ class InstallPackageModal extends mixin(InternalStorageMixin, TabsMixin, StoreMi
     let {isLoading} = this.internalStorage_get();
     let cosmosPackage = CosmosPackagesStore.getPackageDetails();
 
+    // CLI only package
+    if (cosmosPackage && !cosmosPackage.isCLIOnly()) {
```
Comment (jfurrow) [inline src/js/components/modals/InstallPackageModal.js]: Shouldn't this be testing for `isCLIOnly` to be true?

Comment (mlunoe) [inline src/js/components/modals/InstallPackageModal.js]: Yes of course! Done

Comment (jfurrow) [inline src/js/components/modals/InstallPackageModal.js]: LGMT!

**Pull request merged**